pb/demo.proto

```diff
@@ -16,6 +16,8 @@ syntax = "proto3";
 
 package hipstershop;
 
+option go_package = "./;hipstershop";
```

Comment:
Is the semicolon intentional?

Reply:
the meaning of semicolon is described [here](https://developers.google.com/protocol-buffers/docs/reference/go-generated#package). Though I realize it is discouraged to use now, I keep it for now.